pdr_backend/pdr_dashboard/test/test_callbacks_home.py

```diff
@@ -89,8 +89,7 @@ def test_favourite_addresses_search_input(
     fav_addr_toggle.click()
     time.sleep(2)
     p_all, p_sel = _predictoor_count(dash_duo)
-    assert len(p_all) == 58
```

Comment:
I think this will sort itself out once the "storage" is initialised from ppss

Reply:
we can't do that, the local storage can be accessed only trough callbacks